library/src/main/res/layout/mapbox_mapview_internal.xml

```diff
@@ -49,29 +50,62 @@
     <!-- This is the target button shown on the bottom-right -->
     <ImageButton
         android:id="@+id/ib_mapview_focusOnMyLocationIcon"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:src="@drawable/ic_my_location"
-        android:layout_marginBottom="20dp"
+        android:layout_width="70dp"
+        android:layout_height="70dp"
+        android:src="@drawable/ic_cross_hair"
+        android:layout_marginTop="20dp"
         android:layout_marginRight="20dp"
-        android:layout_gravity="bottom|right"
+        android:layout_gravity="top|right"
         android:padding="10dp"
         android:background="@drawable/circle_button"
-        android:layout_alignParentRight="true"
-        android:layout_alignParentBottom="true"
-        android:visibility="gone"/>
+        android:visibility="gone" />
 
     <ImageButton
```

Comment:
I don't think we need this in the library. It doesn't have a function as of now and can be easily overlaid on top of our view by in the host applications layout.

Reply:
I don't think the hamburger menu should be in the library, any reason for having it in the library

Reply:
This is already removed.